messaging/src/main/java/org/axonframework/messaging/eventhandling/processing/streaming/pooled/Coordinator.java

```diff
@@ -318,7 +318,7 @@ private boolean initializeTokenStore() {
                     context -> {
                         List<Segment> segments = joinAndUnwrap(tokenStore.fetchSegments(name, context));
                         if (segments.isEmpty()) {
-                            logger.info("Initializing segments for processor [{}] ({} segments)",
+                            logger.debug("Initializing segments for processor [{}] ({} segments)",
```

Comment:
This one gives a general overview of what happens, I think it is best to leave at info level.

Reply:
This will happen once in the lifecycle of an application. Or, on a clean of you token store every time. It is most likely "spamming" as you're playing around with a clean database.

Hence, I align with @hjohn here. We'll keep this on info for the time being.